appcenter-file-upload-client-node/package.json

```diff
@@ -40,6 +40,7 @@
   },
   "dependencies": {
     "abort-controller": "3.0.0",
-    "node-fetch": "2.6.0"
+    "node-fetch": "2.6.0",
+    "proxy-agent": "^3.1.1"
```

Comment:
Any reason to use `^` caret notation here? We usually stick to fixed version numbers and rely on dependabot to tell us about updates.